web/src/main/java/org/akaza/openclinica/service/UserServiceImpl.java

```diff
@@ -387,6 +388,87 @@ public void extractParticipantsInfo(String studyOid, String siteOid, String acce
         }
         persistJobCompleted(jobDetail, fileName);
     }
+    
+    @Transactional
+    public StudyParticipantDetailDTO extractParticipantInfo(String studyOid, String siteOid, String accessToken, String customerUuid, UserAccountBean userAccountBean, String schema,String participantID,boolean incRelatedInfo) throws OpenClinicaSystemException
+    {
+
+        CoreResources.setRequestSchema(schema);
+
+        Study site = studyDao.findByOcOID(siteOid);
+        Study study = studyDao.findByOcOID(studyOid);
+
+
+        // Get all list of StudySubjects by studyId
+        StudySubject studySubject = studySubjectDao.findByLabelAndStudyOrParentStudy(participantID, study);
+        OCUserDTO ocuserDTO = null;
+        StudyParticipantDetailDTO spDTO= new StudyParticipantDetailDTO();
+        
+        if(studySubject == null) {
+        	String errorCode ="errorCode.participantNotAvailable";
+        	String msg = "Can't find the participant with ID:" + participantID;
+        	throw new OpenClinicaSystemException(errorCode, msg);
+        }
+            
+        try {
+          
+            	if (!studySubject.getStatus().equals(Status.DELETED)
```

Comment:
since this is already implemented in the other method, please refactor this so we can reuse that code.

Reply:
done